content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -181,14 +181,17 @@ A time zone database from the Go standard library is included in the binaries an
 
 ### Unsupported TimeZone specification
 
-The implementation of the CronJob API in Kubernetes {{< skew currentVersion >}} lets you set
+The implementation of the CronJob API in Kubernetes 1.22-1.28 lets you set
 the `.spec.schedule` field to include a timezone; for example: `CRON_TZ=UTC * * * * *`
 or `TZ=UTC * * * * *`.
 
 Specifying a timezone that way is **not officially supported** (and never has been).
 
-If you try to set a schedule that includes `TZ` or `CRON_TZ` timezone specification,
-Kubernetes reports a [warning](/blog/2020/09/03/warnings/) to the client.
+Starting with Kubernetes 1.29 if you try to set a schedule that includes `TZ` or `CRON_TZ`
+timezone specification, Kubernetes will fail to create the resource with a validation
+error.
+Updates to CronJobs already using `TZ` or `CRON_TZ` will continue to report a
+[warning](/blog/2020/09/03/warnings/) to the client.
 Future versions of Kubernetes will prevent setting the unofficial timezone mechanism entirely.
```

Comment:
I think you can drop the "future versions" line now... 1.29 is the future version that has now arrived.

Reply:
Since these are the 1.29 docs, you can remove
>The implementation of the CronJob API in Kubernetes 1.22-1.28 lets you set
the `.spec.schedule` field to include a timezone; for example: `CRON_TZ=UTC * * * * *`
or `TZ=UTC * * * * *`.

> Specifying a timezone that way is **not officially supported** (and never has been).

That section will still be in the 1.24, 1.25, 1.26, 1.27, 1.28 docs